src/test/shared/credentials/loginManager.test.ts

```diff
@@ -66,7 +65,18 @@ describe('LoginManager', async function () {
     })
 
     function assertTelemetry(expected: Parameters<typeof recordAwsValidateCredentials>[0]): void {
```

Comment:
should we extract this to testUtil.ts

Reply:
Yeah I'll clean it up and just handle all the edge cases. Unfortunate that the logger won't be too ergonomic on its own (at least for now) but eventually it'll get there.